pkg/plugins/policies/meshtcproute/api/v1alpha1/meshtcproute.go

```diff
@@ -54,6 +54,7 @@ type To struct {
 type Rule struct {
 	// Default holds routing rules that can be merged with rules from other
 	// policies.
+	// +kuma:non-mergeable-struct
```

Comment:
The `Default RuleConf` is mergeable we shouldn't mark it with `+kuma:non-mergeable-struct`. I think the annotation on `// +kubebuilder:validation:MinItems=1` is incorrect. It should be optional the same way we have in MeshHTTPRoute https://github.com/kumahq/kuma/blob/f153461bd6e33dad7004209969591827604b6732/pkg/plugins/policies/meshhttproute/api/v1alpha1/meshhttproute.go#L107